extensions/0832/SuperAlert.js

```diff
@@ -0,0 +1,317 @@
+        }, 300);
+    }
+
+    class f {
+        constructor() {
+            this.g = "";
+            this.h = "";
+            this.xPos = -50; // Default X position
+            this.yPos = -50; // Default Y position
+            this.titleFont = "Arial";
+            this.contentFont = "Arial";
+            this.titleColor = "white";
+            this.contentColor = "white";
+            this.titleSize = 24;
+            this.contentSize = 16;
+            this.outlineSize = 1;
+        }
+        d() {
+            if (!b) {
+                d();
+                c.style.display = "block";
+                this.i();
+            }
+        }
+
+        e() {
+            if (b) {
+                e();
+            }
+        }
+
+        j(k) {
+            this.g = k.L;
+            this.i();
+        }
+
+        l(k) {
+            this.h = k.M;
+            this.i();
+        }
+
+        p(k) {
+            this.xPos = k.X;
+            this.yPos = k.Y;
+            this.i();
+        }
+
+        setFont(k) {
+            if (k.T === 'Title') {
+                this.titleFont = k.F;
+            } else if (k.T === 'Content') {
+                this.contentFont = k.F;
+            }
+            this.i();
+        }
+
+        setTextColor(k) {
+            if (k.T === 'Title') {
+                this.titleColor = k.C;
+            } else if (k.T === 'Content') {
+                this.contentColor = k.C;
+            }
+            this.i();
+        }
+
+        setTextSize(k) {
+            if (k.T === 'Title') {
+                this.titleSize = k.S;
+            } else if (k.T === 'Content') {
+                this.contentSize = k.S;
+            }
+            this.i();
+        }
+
+        alignTitleText(k) {
+            this.titleAlignment = k.A;
+            this.i();
+        }
+
+        alignContentText(k) {
+            this.contentAlignment = k.A;
+            this.i();
+        }
+
+        i() {
+            if (b) {
+                c.style.opacity = "0";
+                c.style.transform = `translate(${this.xPos}%, ${this.yPos}%) scale(0)`;
+                setTimeout(() => {
+                    c.innerHTML = `
+                <h2 style="margin-bottom: 10px; font-family: ${this.titleFont}; font-size: ${this.titleSize}px; color: ${this.titleColor}; text-shadow: ${this.outlineSize}px ${this.outlineSize}px 0px rgba(0, 0, 0, 0.7); text-align: ${this.titleAlignment};">${this.g}</h2>
+                <p style="font-family: ${this.contentFont}; font-size: ${this.contentSize}px; color: ${this.contentColor}; text-shadow: ${this.outlineSize}px ${this.outlineSize}px 0px rgba(0, 0, 0, 0.7); text-align: ${this.contentAlignment};">${this.h}</p>
+                `;
+                    c.style.opacity = "1";
+                    c.style.transform = `translate(${this.xPos}%, ${this.yPos}%) scale(1)`;
+                }, 300);
+            }
+        }
+
+        getInfo() {
+            return {
+                id: 'tc',
```

Comment:
Extension ID should generally be more descriptive, containing the username of the person who made it and the extension's name. "k12superalert" would be a better ID, for example.